ega-charts/localega/values.yaml

```diff
@@ -52,12 +52,12 @@ persistence:
 # revisionHistory: 3
 
 secrets:
-  cega_creds: ""
-  cega_mq_pass: ""
-  pgp_passphrase: ""
-  shared_pgp_password: ""
-  mq_password: ""
-  postgres_password: ""
+  cega_creds: "cega"
```

Comment:
I would not put any default values here as we do have the https://github.com/NBISweden/LocalEGA-deploy-init for generating them, and i think making the extra effort to generate them is good to understand what configuration should be managed in their "testing | staging | production" environment.

Reply:
There's a lot more than this that will change between environments. What I'm aiming for here is to get new devs up and running as quickly as possible. It's also pretty confusing when there's at least two separate repos that seem to do the same thing, eg `LocalEGA-deploy-k8s` and `LocalEGA-deploy-init`. Which one should be used? Should the `k8s` repo be deprecated? Can we use the artifacts from the init script directly with helm rather than copy pasting values?

Reply:
`LocalEGA-deploy-k8s` does not exist it was renamed to `LocalEGA-deploy-init`

Reply:
No, the `localega` part of the helm chart shall be production targeted an thus will not have anything hardcoded.